src/controller/OpenRobertaController.vala

```diff
@@ -75,6 +75,7 @@ namespace BrickManager {
                     });
                 }, () => {
                     status_bar_item.connected = false;
+                    open_roberta_window.connected = false;
```

Comment:
`open_roberta_window` could be null here. Also the `available` property overrides this in a way. If the service is not available, the `open_roberta_window` will display "not availible" regardless of the state of the `connected` property. I was hoping that OpenRoberta/robertalab-ev3dev#2 would be fixed, then connected state won't matter here. Until this, I guess we need this, but it needs a null check.